### ISSUE TITLE
Import-DbaCsvToSql Locking the csv file

### ISSUE THREAD

**Issue by jrob24** [sqlcollaborative/dbatools#3866]
Running version 0.9.382 and after I use Import-DbaCsvToSql the csv file is locked until the powershell console is closed. Is this to be expected? I need to clean up the csv file after import but unable too since the file is locked.

**Comment by jrob24** [comment]
Ok I think I found where the issue is occurring. In the end block lines 1309-1311, the $oleconn.close and $reader.close methods are both missing ( )'s so I added those. Also if you dont use Safe mode then the $oleconn and $oldcmd connections will never get created. Therefore they will throw an error and never execute the $reader.close() method leaving the file locked. I believe some if statements around the $oleconn and $oldcmd connections in the End block would resolve this bug, but there could absolutely be a better way.
